src/Conversion/ONNXToTorch/NN/SqueezeOp.cpp

```diff
@@ -0,0 +1,161 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===------- SqueezeOp.cpp - ONNX Op Transform ------------------===//
+//
+// Copyright 2022, Helprack LLC.
+//
+// =============================================================================
+//
+// This file lowers squeeze operator from onnx to torch using a template for
+// v11 and v13.
+//
+//===----------------------------------------------------------------------===//
+
+#include "src/Conversion/ONNXToTorch/ONNXToTorchCommon.hpp"
+#include "src/Conversion/ONNXToTorch/NN/CommonUtils.h"
+
+#include <cstdint>
+#include <set>
+#include <vector>
+
+using namespace mlir;
+using namespace mlir::torch;
+using namespace mlir::torch::Torch;
+
+std::vector<int> toUniqueAndNonNegative(std::vector<int> axes) {
+  std::set<int> axesSet(axes.begin(), axes.end());
+  std::vector<int> axesNonNeg;
```

Comment:
`std::set` is sorted by default. You can probably remove the reordering lateron.

Reply:
Done